modules/dr-restoring-cluster-state.adoc

```diff
@@ -465,125 +465,17 @@ clustername-8qw5l-worker-us-east-1c-pkg26   Running   m4.large    us-east-1   us
-    instanceId: i-0fdb85790d76d0c3f
-    instanceState: stopped
-    kind: AWSMachineProviderStatus
-----
-
-... Change the `metadata.name` field to a new name by running:
-+
-It is recommended to keep the same base name as the old machine and change the ending number to the next available number. In this example, `clustername-8qw5l-master-0` is changed to `clustername-8qw5l-master-3`:
-+
-[source,terminal]
-----
-apiVersion: machine.openshift.io/v1beta1
-kind: Machine
-metadata:
-  ...
-  name: clustername-8qw5l-master-3
-  ...
-----
-
-... Remove the `spec.providerID` field by running:
-+
-[source,terminal]
-----
-providerID: aws:///us-east-1a/i-0fdb85790d76d0c3f
-----
-
-... Remove the `metadata.annotations` and `metadata.generation` fields by running:
-+
-[source,terminal]
-----
-annotations:
-  machine.openshift.io/instance-state: running
-...
-generation: 2
-----
-
-... Remove the `metadata.resourceVersion` and `metadata.uid` fields by running:
-+
-[source,terminal]
-----
-resourceVersion: "13291"
-uid: a282eb70-40a2-4e89-8009-d05dd420d31a
-----
-
 .. Delete the machine of the lost control plane host by running:
 +
 [source,terminal]
 ----
 $ oc delete machine -n openshift-machine-api clustername-8qw5l-master-0 <1>
 ----
 <1> Specify the name of the control plane machine for the lost control plane host.
-
-.. Verify that the machine was deleted by running:
-+
-[source,terminal]
-----
-$ oc get machines -n openshift-machine-api -o wide
-----
-+
-Example output:
 +
-[source,terminal]
-----
-NAME                                        PHASE     TYPE        REGION      ZONE         AGE     NODE                           PROVIDERID                              STATE
-clustername-8qw5l-master-1                  Running   m4.xlarge   us-east-1   us-east-1b   3h37m   ip-10-0-143-125.ec2.internal   aws:///us-east-1b/i-096c349b700a19631   running
-clustername-8qw5l-master-2                  Running   m4.xlarge   us-east-1   us-east-1c   3h37m   ip-10-0-154-194.ec2.internal   aws:///us-east-1c/i-02626f1dba9ed5bba  running
-clustername-8qw5l-worker-us-east-1a-wbtgd   Running   m4.large    us-east-1   us-east-1a   3h28m   ip-10-0-129-226.ec2.internal   aws:///us-east-1a/i-010ef6279b4662ced   running
-clustername-8qw5l-worker-us-east-1b-lrdxb   Running   m4.large    us-east-1   us-east-1b   3h28m   ip-10-0-144-248.ec2.internal   aws:///us-east-1b/i-0cb45ac45a166173b   running
-clustername-8qw5l-worker-us-east-1c-pkg26   Running   m4.large    us-east-1   us-east-1c   3h28m   ip-10-0-170-181.ec2.internal   aws:///us-east-1c/i-06861c00007751b0a   running
-----
-
-.. Create a machine by using the `new-master-machine.yaml` file by running:
-+
-[source,terminal]
-----
-$ oc apply -f new-master-machine.yaml
-----
+A new machine is automatically provisioned after deleting the machine of the lost control plane host.
 
-.. Verify that the new machine has been created by running:
+.. Verify that a new machine has been created by running:
```

Comment:
```suggestion
.. Verify that a new machine has been created by running the following command:
```
Suggestion per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command - but if that would make it inconsistent with the phrasing in the other steps, it might be best to leave alone. Your call.

Reply:
I noticed that while working on this, too, but kept it for consistency. I might create a new issue to change that phrasing throughout this module.

Thanks!